.github/workflows/kuksa_databroker-cli_build.yml

```diff
@@ -124,31 +124,31 @@ jobs:
         file: ./kuksa_databroker/Dockerfile-cli
         context: .
         push: true
-        tags: "ttl.sh/kuksa.val/kuksa-databroker-cli-${{github.sha}}"
+        tags: "ttl.sh/kuksa.val/kuksa-cli-${{github.sha}}"
         labels: ${{ steps.meta.outputs.labels }}
 
 
     - name: Posting message
       uses: ./.github/actions/post-container-location
       with:
-        image: ttl.sh/kuksa.val/kuksa-databroker-cli-${{github.sha}}
+        image: ttl.sh/kuksa.val/kuksa-cli-${{github.sha}}
 
     - name: "Archiving AMD64 artifacts"
       uses: actions/upload-artifact@v3
       with:
-        name: databroker-cli-amd64
+        name: kuksa-cli-amd64
         path: ${{github.workspace}}/dist/amd64
 
     - name: "Archiving ARM64 artifacts"
       uses: actions/upload-artifact@v3
       with:
-        name: databroker-cli-arm64
+        name: kuksa-cli-arm64
         path: ${{github.workspace}}/dist/arm64
 
     - name: "Archiving RISCV64 artifacts"
       uses: actions/upload-artifact@v3
       with:
-        name: databroker--cli-riscv64
+        name: kuksa--cli-riscv64
```

Comment:
I notice that we since before has `--cli` here while other architectures have `-cli`. Maybe worth aligning when we anyway change name?